x-pack/legacy/plugins/apm/public/components/app/ServiceMap/Cytoscape.tsx

```diff
@@ -62,17 +62,30 @@ export function Cytoscape({
   // Trigger a custom "data" event when data changes
   useEffect(() => {
     if (cy) {
+      cy.remove(cy.nodes());
       cy.add(elements);
       cy.trigger('data');
     }
   }, [cy, elements]);
 
+  useEffect(() => {
```

Comment:
This should be removed, since clicking on a node will bring up a popover.

Reply:
removed